Header header
float64 min_x
float64 max_x
float64 min_y
float64 max_y
float64 min_z
float64 max_z
float64 slope

